python/src/spark_rapids_ml/classification.py

```diff
@@ -1277,6 +1277,9 @@ def _supportsTransformEvaluate(self, evaluator: Evaluator) -> bool:
 
         return False
 
+    def _support_gpuMemRatioForData(self) -> bool:
```

Comment:
Any other algos this could work for?

Reply:
Yes, I think so, will work on supporting more algos after this PR merged.